wuvt/trackman/lib.py

```diff
@@ -256,7 +247,12 @@ def delete_track(track):
                 {TrackLog.track_id: track_id}, synchronize_session=False)
 
             # delete existing Track entries
-            map(delete_track, tracks[1:])
+            for track in tracks[1:]:
```

Comment:
any reason for simplifying the track deduplication here?

Reply:
Yeah, 2to3 suggested doing this when I ran it. I figured it was more clear, so why not?